web/src/common/Header.tsx

```diff
@@ -32,13 +29,17 @@ const NoBulletUnorderedList = styled.ul`
 `
 
 export const Header = (): JSX.Element => {
+  const accountRefElement = useRef<HTMLButtonElement>(null)
+  const infoRefElement = useRef<HTMLButtonElement>(null)
+
   const [isUserInfoOpen, setUserInfoOpen] = useState(false)
   const [isAboutOpen, setAboutOpen] = useState(false)
 
-  const { tokenData, logOut } = useContext(AuthContext)
+  useEffect(() => {
+    setAboutOpen(true)
```

Comment:
Since it uses infoRefElement which is initially null, so the info will open in the top-left corner the split second before the ref is set to anchor at the correct component.